### ISSUE TITLE
MT report access

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#3093]
Given the recent server move in Solna, this is likely a servers issue, but I'll post it here for now:
```
2022-01-19 11:28:44,461 - scout.server.app - ERROR: Exception on /cust003/21323/mt_report [GET] [in /venv/lib/python3.8/site-packages/flask/app.py:1457]
Traceback (most recent call last):
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
   response = self.full_dispatch_request()
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
   rv = self.handle_user_exception(e)
 File "/venv/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
   return cors_after_request(app.make_response(f(*args, **kwargs)))
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
   rv = self.dispatch_request()
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
   return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
 File "/home/worker/app/scout/server/blueprints/cases/views.py", line 251, in mt_report
   os.makedirs(temp_excel_dir, exist_ok=True)
 File "/usr/local/lib/python3.8/os.py", line 223, in makedirs
   mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/worker/app/scout/server/blueprints/cases/static/21323_mt_reports'
```

**Comment by dnil** [comment]
It replicates on the stage side, so we can test there.
```
2022-01-19 11:33:28,496 - scout.server.app - ERROR: Exception on /cust003/15001/mt_report [GET] [in /venv/lib/python3.8/site-packages/flask/app.py:1457]
Traceback (most recent call last):
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
   response = self.full_dispatch_request()
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
   rv = self.handle_user_exception(e)
 File "/venv/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
   return cors_after_request(app.make_response(f(*args, **kwargs)))
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
   rv = self.dispatch_request()
 File "/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
   return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
 File "/home/worker/app/scout/server/blueprints/cases/views.py", line 251, in mt_report
   os.makedirs(temp_excel_dir, exist_ok=True)
 File "/usr/local/lib/python3.8/os.py", line 223, in makedirs
   mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/worker/app/scout/server/blueprints/cases/static/15001_mt_reports'
```

**Comment by northwestwitch** [comment]
Fixed by #3095
